src/components/Header.vue

```diff
@@ -5,38 +5,41 @@
         <router-link to="/">
           <img class="logo rotating" alt="Mesh logo" src="../assets/mesh-logo.png">
         </router-link>
+        <h1 class="s-h1 greeting">Mesh</h1>
       </div>
 
-      <h2 class="date">
-        <!--          <img class="calendar" alt="Calendar icon" src="../assets/calendar.svg">-->
-        <a class="outside-link" href="https://frontend.turing.edu/today/" target="_blank">
-          <i class="calendar fal fa-calendar-alt"></i>
-          {{ date }}
-        </a>
-      </h2>
+      <div>
+        <h3 class="name">{{ user.userId === 112 ? 'Instructor' : 'Student' }}</h3>
```

Comment:
Idea: Instead of just "Instructor" or "Student", this could say "Instructor Dashboard" or "Student Dashboard"